packages/bootleg-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
The root .gitignore I think covers most of these?

Reply:
autogenerated by `create-react` I'll remove.